src/main/resources/root.jsp

```diff
@@ -17,6 +17,23 @@
 
     <internal:gwtGenerateDictionary/>
     <internal:gwtInit/>
+
+    <script type="text/javascript">
+        Object.assign(window.contextJsParameters, {
+            version: '<%= Jahia.VERSION %>',
+            contextPath: '${contextPath}',
+            user: {
+                username: '${currentUser.name}',
+                fullname: '${user:fullName(currentUser)}',
+                email: '${currentUser.properties['j:email']}',
+                path: '${currentUser.localPath}'
+            },
+            config: ${config},
+            siteKey: '${defaultSite.siteKey}',
+            siteUuid: '${defaultSite.identifier}',
```

Comment:
what do you need the siteUuid for ?

Reply:
for GWT to set the site correctly

Reply:
the site should anyway be set by the redux listener